src/users/users-service.js

```diff
@@ -0,0 +1,49 @@
+const bcrypt = require('bcryptjs')
+const xss = require('xss')
+
+const REGEX_UPPER_LOWER_NUMBER_SPECIAL = /(?=.*[a-z])(?=.*[A-Z])(?=.*[0-9])(?=.*[!@#\$%\^&])[\S]+/
+
+const UsersService = {
+  hasUserWithUserName(db, email) {
+    return db('users')
+      .where({ email })
+      .first()
+      .then(user => !!user)
+  },
+  insertUser(db, newUser) {
+    return db
+      .insert(newUser)
+      .into('users')
+      .returning('*')
+      .then(([user]) => user)
+  },
+  validatePassword(password) {
+    if (password.length < 8) {
+      return 'Password be longer than 8 characters'
+    }
+    if (password.length > 72) {
+      return 'Password be less than 72 characters'
```

Comment:
"Passwords _must_ be longer than 8 characters' 
"Passwords _must_ be less than 8 characters'

Reply:
change has been mande